### DISCUSSION TITLE
Tech coop popularity in the United States

### DISCUSSION THREAD

**Discussion by obsidiesque** [hng/tech-coops#172] [General]
I started looking into tech coops after having ran into [this post on coops](https://medium.com/camplight/fired-why-cooperatives-might-be-your-next-career-choice-in-tech-73d689466609) posted on HackerNews

I couldn't help to wonder why coops aren't as popular in the United States as it is in Europe. I think practically all if not all the members on [coops.tech](https://www.coops.tech/#members) are in Europe.

As a knowledge worker it seems like a pretty good type of organization that rewards you with ownership for your contributions and an actual sense of community.

Beyond tech coops, I wonder why this isn't a more popular structure for startups, but I think the answer there might be VC capital.

Would love to hear thoughts

**Comment by robinkeunen** [comment]
Hi there and welcome,

Most members of coops.tech are actually in the UK, it's a [criterium to join in](https://wiki.coops.tech/wiki/Who_can_join_CoTech).

The principles of a cooperative is that its beneficiary are also the owners of the company. In a worker cooperative\*, the workers are the owners and as such control the company. Additionally, you usually   have a rule of _one person gets one vote_ in the general assembly rather than _one share gets one vote_.

So yeah, a VC investor wants to pump in a lot of money, get some control over the board and sell the shares when the company value grew. You can't get any of that in a "proper" worker cooperative.

More over, some cooperatives do not allow share reevaluation. Therefore you get out of the coop with what you put in. The surplus cooperative value stays in the cooperative to serve its social finality.

\* other cooperatives can be 
- customer cooperatives in which the customer controls the company that provide good to them ([Park Slope Coop](https://www.foodcoop.com) or the historic [Rochdale Society of Equitable Pioneers](https://en.wikipedia.org/wiki/Rochdale_Society_of_Equitable_Pioneers)
- producer cooperative in which the producers controls the company that provide common infrastructure and secure a bigger control of the market (usually in agriculture).
- credit / bank cooperatives where the customer controls the investment politics ([NewB](https://www.newb.coop/nl), [Credal](https://www.credal.be) in Belgium)

**Comment by carlos-reynosa** [comment]
From what i've looked into, laws and the financial culture can have an impact on the formation of tech coops. Certain laws are built to only recognize certain types of corporations. For example maybe states will recognize only one owner as apposed to multiple owners found in cooperatives. That in turn affects the way that financial institutions recognize and are willing to work with Cooperatives. For example who is liable for debts and is recognize as account owners.  That in turn affects funding and the ability to start coops. 

Some links:

* https://www.youtube.com/watch?v=87395oUPDR0
* https://www.youtube.com/watch?v=vy3bbzTHDnU&t=3s
